src/containers/Notifications.js

```diff
@@ -20,7 +20,59 @@ export class Notifications extends Component {
   }
 
   render() {
-    return <NotificationSystem ref="notificationSystem" />;
+    const style = {
```

Comment:
We don't want this here. It will be set in every render. We can move it outside the class or maybe another file.

Reply:
hmm.. valid point. How about making this a class property?
Would that make this static for every *instance*..?

Reply:
Yeah, that would work.